boards/arm/nrf52/nrf52840-dk/src/nrf52_highpri.c

```diff
@@ -96,6 +96,11 @@ static struct highpri_s g_highpri;
  * Private Functions
  ****************************************************************************/
 
+static inline_function bool parent_in_interrupt(void)
```

Comment:
why not replace to up_current_regs() directly?

Reply:
maybe up_current_regs is not a clear name here, we have two choice:
1. use up_current_regs directly, and add some comment here, to make next  maintainer less confused.
2. use a statuc inline function parent_in_interrupt to indicate what we want, and if futher patch changed the up_current_regs behavior, should also update the function inside parent_in_interrupt.

personally prefer 2nd method,

Reply:
Ok, could we switch a common naming like `is_nesting_interrupt()` instead of `parent_in_interrupt()`?